tests/e2e/hcp_machine_pool_test.go

```diff
@@ -762,6 +762,78 @@ var _ = Describe("HCP MachinePool", ci.Day2, ci.FeatureMachinepool, func() {
 
 		})
 
+	It("can create machinepool with disk size - [id:69144]", ci.High, func() {
```

Comment:
It's a critical case a/c to the polarion test case

Reply:
Also a/c to test case it can be run on all sts cluster, but here it will run only on hcp cluster

Reply:
The case 69144 is already automated in classic machine pool testing.
Would there be a possibility to reuse it ? Or do you prefer to have 2 separate code ?

Reply:
But actually test case mentions the product to be both ROSA and ROSA HCP , that's why I was confused
I guess test case needs to be updated for ROSA HCP only

Reply:
> It's a critical case a/c to the polarion test case
Hmm, let me update both classical and hcp case to critical, which keep the same with polarion cases.

Reply:
> The case 69144 is already automated in classic machine pool testing.
> Would there be a possibility to reuse it ? Or do you prefer to have 2 separate code ?

Hmm, I think it's better to have 2 separate code for two reasons: (1) the machine pool structures of hcp and classical are different, the warning message are also different(e.g. For classical, `Attribute disk_size, cannot be changed from xxx to yyy`. For HCP, `Attribute aws_node_pool.disk_size, cannot be changed from xxx to yyy`), which all require special handling. (2) The disk size of hcp and classical have different minimum.

Reply:
Good idea. I've move test case `69144`, `76345` to a new test file `machine_pool_test.go`.